data/products.json

```diff
@@ -3,193 +3,526 @@
     "name": "Adobe Photoshop",
     "country_code": "US",
     "alternatives": [
-      {
```

Comment:
To make the pull request easier to review, let's try to maintain the original order of the products.